networking_cisco/plugins/cisco/device_manager/plugging_drivers/aci_vlan_trunking_driver.py

```diff
@@ -226,7 +227,7 @@ def _add_snat_info(self, context, router, net, hosting_info):
             if snat_subnets and snat_ips:
                 hosting_info['snat_subnets'] = []
                 for subnet in snat_subnets:
-                    snat_subnet = {'id': subnet['id'],
+                    snat_subnet = {'id': router['tenant_id'],
```

Comment:
The first requirement is to support an SNAT pool per project in OpenStack. The initial implementation did not meet that requirement. Future versions will either get this ID from the mechanism drivers, or implement some strategy based on a config in an ASR config file (need to get decision on where this configuration should live first).